screens/Todos.tsx

```diff
@@ -1,17 +1,35 @@
 import React from 'react';
 import { View, StyleSheet } from 'react-native';
 
+import {
+  NavigationParams,
+  NavigationScreenProp,
+  NavigationState,
+} from 'react-navigation';
+
 import List from '../components/Todo/List';
 
-const TodosScreen = () => (
-  <View style={styles.container}>
-    <View style={styles.content}>
-      <List />
+interface TodosNavOptions {
+  navigationOptions: {
+    title: String;
+  };
+}
+
+interface TodosProps {
+  navigation: NavigationScreenProp<NavigationState, NavigationParams>;
+}
+
+const Todos: React.SFC<TodosProps> & TodosNavOptions = () => {
```

Comment:
OK, I found a solution for typing functional component, here:
https://reactnavigation.org/docs/en/typescript.html#type-checking-screen-components
So, it will be just:
`const Todos: NavigationStackScreenComponent = () => {`

Reply:
Just official docs :D Corrected, should be okey now